apps/src/templates/instructions/InlineFeedback.jsx

```diff
@@ -26,10 +26,9 @@ class InlineFeedback extends Component {
     // styling.
     return (
       <ChatBubble borderColor={borderColor} ttsMessage={message}>
-        <p
-          className="uitest-topInstructions-inline-feedback"
-          dangerouslySetInnerHTML={{__html: message}}
-        />
+        <div className="uitest-topInstructions-inline-feedback">
```

Comment:
This uitest class shouldn't be required anymore now that you have UnsafeRenderedMarkdown as a react component

Reply:
How so? It's still used by our UI tests

Reply:
Sorry for the vague comment. I was suggesting a quick refactor so we don't have test code mixed in with our product code. Now that you've made this change, we no longer need it mixed in like we did before.

Reply:
i.e. update our tests to reference UnsafeRenderedMarkdown rather than uitest-topInstructions-inline-feedback

Reply:
Ah, gotcha.

It certainly would be nice to not have test code in here, but unfortunately the UI tests run at the browser level, not at the code level, and so aren't easily able to detect React components on their own

Reply:
Offline conversation: We agree that it would be good to investigate a UI test infrastructure that doesn't require adding div IDs just for testing, but there isn't a good way to make this work in the current infrastructure without it. Also, it's far outside the scope of this PR.